tutorials/tagger/basic_allennlp.py

```diff
@@ -205,3 +205,45 @@ def get_metrics(self, reset: bool = False) -> Dict[str, float]:
 tag_ids = np.argmax(tag_logits, axis=-1)
 #### And then use our vocabulary to find the predicted tags.
 print([model.vocab.get_token_from_index(i, 'labels') for i in tag_ids])
+
+#### Finally, we'd like to be able to save our model and reload it later. We'll need to save two things. The first is the model weights.
```

Comment:
Minor thought, not for this PR: is it possible to do something like combine contiguous blocks of lines beginning with `####` in the `convert.py` script, so you don't have these super long lines?  It'd make reading and reviewing these files a lot easier.